packages/sdks/src/blocks/image/image.helpers.ts

```diff
@@ -58,7 +58,7 @@ export function getSrcSet(url: string): string {
     return sizes
       .filter((size) => size !== widthInSrc)
       .map((size) => `${updateQueryParam(url, 'width', size)} ${size}w`)
-      .concat([srcUrl])
+      .concat([`${srcUrl} 9999w`])
```

Comment:
In the few lines above this one, you see that we do set the width when we have it:

https://github.com/BuilderIO/builder/blob/75bc0d0017d99cd13682263aefb74b94df682d2d/packages/sdks/src/blocks/image/image.helpers.ts#L52-L62


So a better place to be is to set the `9999w` value in an `else` condition there:

```ts
    if (!isNaN(widthInSrc)) {
      srcUrl = `${srcUrl} ${widthInSrc}w`;
    } else {
      srcUrl = `${srcUrl} 9999w`;
    }
```